### ISSUE TITLE
Can anybody tell me the paper explaining this project 

### ISSUE THREAD

**Issue by RUPESHKUMARPALLAI** [ZhaoJ9014/face.evoLVe.PyTorch#162]
Dear authors,
               Do u have any resources explaining loss function working and model training?
Thanks in advance.

**Comment by RUPESHKUMARPALLAI** [comment]
Urgent please

**Comment by perfectZh** [comment]
@article{wang2021face,
title={Face. evoLVe: A High-Performance Face Recognition Library},
author={Wang, Qingzhong and Zhang, Pengfei and Xiong, Haoyi and Zhao, Jian},
journal={arXiv preprint arXiv:2107.08621},
year={2021}
}
